tests/containers/apptainer.pm

```diff
@@ -14,11 +14,11 @@ use containers::utils qw(registry_url);
 
 sub run {
     my ($self) = @_;
-    # Not used of serial_terminal because causes some delays due to wait_serial
-    # failing match on `serial_term_prompt`
-    select_console 'root-console';
+    $self->select_serial_terminal;
     my $image = 'registry.opensuse.org/opensuse/tumbleweed:latest';
     my $registry = registry_url() . "/library";
+
+    assert_script_run "systemctl stop tmp.mount";
```

Comment:
Must be undone, otherwise this invalidates all succeeding modules.

Reply:
Fixed with APPTAINER_TMPDIR environment variable.